ground/gcs/src/inc/widgetnames.h

```diff
@@ -1,6 +1,18 @@
 #ifndef WIDGETNAMES_H
 #define WIDGETNAMES_H
 
-
+#define CONFIGGADGETWIDGET_SIDEBAR						"configGadgetSideBar"
+#define CONFIGGADGETWIDGET_SIDEBAR_VBOXLAYOUT			"configGadgetSideBarVBox"
+#define CONFIGGADGETWIDGET_HWSETTINGS					"configGadgetHwSettings"
+#define CONFIGGADGETWIDGET_VEHICLETYPE					"configGadgetVehicleType"
+#define CONFIGGADGETWIDGET_INPUT						"configGadgetInput"
+#define CONFIGGADGETWIDGET_OUTPUT						"configGadgetOutput"
+#define CONFIGGADGETWIDGET_ATTITUDE						"configGadgetAttitude"
+#define CONFIGGADGETWIDGET_STABILIZATION				"configGadgetStabilization"
+#define CONFIGGADGETWIDGET_MODULES						"configGadgetModules"
+#define CONFIGGADGETWIDGET_AUTOTUNE						"configGadgetAutotune"
+#define CONFIGGADGETWIDGET_CAMERASTAB					"configGadgetCameraStab"
+#define CONFIGGADGETWIDGET_TXPID						"configGadgetTxPID"
+#define CONFIGGADGETWIDGET_OSD							"configGadgetCameraOSD"
 
```

Comment:
I can stick literals directly into the files, if ideally we don't intend to rename them in bulk anytime soon. The work is almost the same.